extension/src/Extension.ts

```diff
@@ -315,8 +317,13 @@ export class Extension {
   private handleWatchEvents(): void {
     this.buildFileWatcher.onDidChange(async (uri: vscode.Uri) => {
       logger.info('Build file changed:', uri.fsPath);
+      void this.syncBuildFile(uri);
```

Comment:
The cache will be hit and return an old project content to language server. 
But on file save, the cache actually should expire and refetch project content, right?

Reply:
Yes. Currently we have https://github.com/microsoft/vscode-gradle/blob/6f051190b7273a2e747555aa100e09dbab4e80dc/extension/src/Extension.ts#L318 here to refresh all the caches including project content caches. I'll change this `syncBuildFile()` to after the `refresh()` to let's sync with the new content.

Reply:
reference: https://github.com/microsoft/vscode-gradle/blob/6f051190b7273a2e747555aa100e09dbab4e80dc/extension/src/commands/RefreshCommand.ts#L23